src/components/transport_manager/include/transport_manager/transport_adapter/threaded_socket_connection.h

```diff
@@ -74,6 +74,11 @@ class ThreadedSocketConnection : public Connection {
    */
   TransportAdapter::Error Disconnect();
 
+  /**
+   * @brief Terminate the current connection.
+   */
+  void Terminate() OVERRIDE;
```

Comment:
@SNiukalov remove description from override

Reply:
@alexkutsan 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/3134/commits/53e3d5496660cab96c95bc323d30cbfa23bb1895